test/test_torch.py

```diff
@@ -9659,6 +9659,21 @@ def add_neg_dim_tests():
         assert not hasattr(TestTorch, test_name), "Duplicated test name: " + test_name
         setattr(TestTorch, test_name, make_neg_dim_test(name, tensor_arg, arg_constr, types, extra_dim))
 
+class TestRPATH(TestCase):
+    @unittest.skipIf(not sys.platform.startswith('linux'), "linux-only test")
```

Comment:
Don't you need to guard it on `@onlyCUDA` as well? As CPU binaries should not have libcaffe2_nvrtc.so?

Reply:
`@onlyCUDA` runs this if the test is run on GPU, not if PyTorch is build with CUDA if I understand that correctly.

Hence the check if that library exists below and just run this always.